Attendance/src/main/java/com/system/Attendance/domain/Location.java

```diff
@@ -2,22 +2,32 @@
 
 import com.system.Attendance.enums.LocationType;
 import jakarta.persistence.*;
-import lombok.Data;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 import java.io.Serializable;
 
 @Entity
-@Data
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
oh use this anotation will auto generate getter, setter and constructor for us ?